extra/smtp-testing/smtp.mock.yml

```diff
@@ -7,8 +7,8 @@ services:
             - PYTHONUNBUFFERED=1
         command: bash -c "mkdir -p /var/spool/mail; stdbuf -oL python -m smtpd -n -c DebuggingServer 0.0.0.0:1025 > /var/spool/mail/local"
         ports:
-            - 1025:1025
+            - "1025"
```

Comment:
ah right! it was a copy-paste from a setup where I accessed it outside of the "local" network.

Reply:
But this alone didn't fix the issue, as the `networks` below was incorrect. I couldn't find an exact reference in the Docker compose documentation, but it seems like in the top level you use `mender:` and it is enough but here for the services we require `-mender`.